packages/richtext-slate/src/index.ts

```diff
@@ -52,6 +61,16 @@ export function slateEditor(
   }
 }
 
+function withNullableType(
```

Comment:
Can we import this from Payload instead of having it written in multiple packages?

Reply:
Do you have a suggestion for the naming of this function? I think just exporting `withNullableType` might be confusing if the context is missing